base-theme/config.toml

```diff
@@ -0,0 +1,11 @@
+baseurl = ""
+languageCode = "en-us"
+languageLang = "en"
+title = "MIT OpenCourseWare"
+relativeURLs = false
+enableRobotsTXT = true
+
+# RSS, categories and tags disabled for an easy start
+# See configuration options for more details:
+# https://gohugo.io/getting-started/configuration/#toml-configuration
+disableKinds = ["RSS", "taxonomy", "taxonomyTerm"]
```

Comment:
There were some configuration options to adjust cache `getJSON` calls so that we have new news items fetched when builds happen. Why are those options missing here? https://github.com/mitodl/ocw-www/blob/main/site/config.toml#L13

Reply:
I seemed to have missed those when I was re-writing the config files so I'll add them back.  I think they belong in `ocw-www`'s configuration though and not the theme.

Reply:
> There's a race condition which happens when `npm start` is run, where hugo and webpack are started at the same time but hugo is looking for webpack files to create links for the javascript files. These files exist the second time hugo is started but they refer to the files created the first time webpack was started. I'm not sure it's worth holding up the PR for so I'll file an issue

Yea, I've experienced this and wasn't able to find a quick and easy solution but I think it's worth looking into outside of the scope of this PR.